capture/configs/ibex_sca_chip.yaml

```diff
@@ -7,25 +7,46 @@ target:
   output_len_bytes: 16
   protocol: "ujson"
   port: "/dev/ttyUSB1"
+  usb_serial: "205F355C3236"
```

Comment:
Is this intentional? I guess it will not work in other people's setups right?

Reply:
Fixed, thanks.